buildSrc/src/main/groovy/hale.groovy-conventions.gradle

```diff
@@ -0,0 +1,20 @@
+/*
+ * A library part of hale-core.
+ */
+
+plugins {
+  id 'groovy'
+  id 'hale.java-conventions'
+}
+
+spotless {
+  groovy {
+    importOrder()
```

Comment:
While not restoring original behaviour, we could use `importOrder('groovy', 'java', 'javax', 'org', 'com', '')` here to align it with the Java import order I suggested below.

Reply:
Sure, fine for me.